bin/version.json

```diff
@@ -10,6 +10,6 @@
     "next": "1.11.1"
   },
   "versionCut": {
-    "is_cut": "false"
+    "is_cut": "true"
```

Comment:
The version is not cut, the point of Step 3 is to build distros everyday with the latest version on S3.
It only gets cut by the end of Sanity test when devs sign off the versions for final candidate.